### PR TITLE
Create a rg-tag component

### PR DESCRIPTION
**Pull request by darwinlopez**
Some CSS tweaks and refactoring might be needed

### CODE REVIEW

File: src/rg-tag/rg-tag.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.rgTag {
+  background: var(--color-green-base);
+  -webkit-border-radius: 16;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: should be moved to a variable also the should only need `border-radius`

File: src/rg-tag/rg-tag.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  position: relative;
+  cursor: pointer;
+}
+
+.rgTag:hover {
+  background: var(--color-red-base);
+  text-decoration: none;
+}
+
+.rgTagRemoveButton {
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: `.rgTag--removeButton`

File: src/rg-tag/rg-tag.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    background: var(--color-green-hover);
+    display: inline-block;
+    -webkit-box-sizing: content-box;
+    -moz-box-sizing: content-box;
+    box-sizing: content-box;
+    width: 1.25em;
+    height: 1.25em;
+    position: relative;
+    border: none;
+    -webkit-border-radius: 1.25em;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: autoprefixer

File: src/rg-tag/rg-tag.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+.rgTag:hover {
+  background: var(--color-red-base);
+  text-decoration: none;
+}
+
+.rgTagRemoveButton {
+    background: var(--color-green-hover);
+    display: inline-block;
+    -webkit-box-sizing: content-box;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: autoprefixer

File: src/rg-tag/rg-tag.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
+.rgTag {
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: /** @define rgTag; use strict */

should be at the top

File: src/rg-tag/rg-tag.css [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  background: var(--color-green-base);
+  -webkit-border-radius: 16;
+  -moz-border-radius: 16;
+  border-radius: 16px;
+  font-family: Arial;
+  color: #ffffff;
+  font-size: 14px;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: font-sizes should use the variables set such as badge https://github.com/rockabox/rbx_ui_components/blob/master/src/rb-badge/rb-badge.css#L35

File: src/rg-tag/rg-tag.css [hunk 7]
```diff
@@ -0,0 +1,48 @@
+.rgTag {
+  background: var(--color-green-base);
+  -webkit-border-radius: 16;
+  -moz-border-radius: 16;
+  border-radius: 16px;
+  font-family: Arial;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: font-family should simply inherit from the site

File: src/rg-tag/rg-tag.css [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    -webkit-box-sizing: content-box;
+    -moz-box-sizing: content-box;
+    box-sizing: content-box;
+    width: 1.25em;
+    height: 1.25em;
+    position: relative;
+    border: none;
+    -webkit-border-radius: 1.25em;
+    border-radius: 1.25em;
+    font: normal 1em/normal Arial, Helvetica, sans-serif;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: font family should simply inherit from the site

File: src/rg-tag/rg-tag.css [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    color: var(--color-white-catskill);
+    -o-text-overflow: clip;
+    text-overflow: clip;
+}
+
+.rgTagDisabled {
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: `.rgTag--disabled`

File: src/rg-tag/rg-tag.css [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  -webkit-border-radius: 16;
+  -moz-border-radius: 16;
+  border-radius: 16px;
+  font-family: Arial;
+  color: #ffffff;
+  font-size: 14px;
+  margin: 8px 8px 0 0;
+  padding: 0px 12px 0px 12px;
+  text-decoration: none;
+  cursor: default;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: should this not be `cursor: pointer`?

File: test/unit/rg-tag/rg-tag.spec.js [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+                    element = $compile(tag)($rootScope.$new()),
+                    removeButton;
+
+                $rootScope.$apply();
+
+                removeButton = angular.element(element.find('button'))[0];
+                expect(removeButton).toBeUndefined();
+            }));
+
+        });
```
Comment (nwhite89) [inline test/unit/rg-tag/rg-tag.spec.js]: test for removing of tags?

File: src/rg-tag/rg-tag.css [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+    width: 1.25em;
+    height: 1.25em;
+    position: relative;
+    border: none;
+    -webkit-border-radius: 1.25em;
+    border-radius: 1.25em;
+    font: normal 1em/normal Arial, Helvetica, sans-serif;
+    color: var(--color-white-catskill);
+    -o-text-overflow: clip;
+    text-overflow: clip;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: autoprefixer should take care of this

File: test/unit/rg-tag/rg-tag.spec.js (hunk 11 above)
Comment (darwinlopez) [inline test/unit/rg-tag/rg-tag.spec.js]: Done

File: src/rg-tag/rg-tag.css (hunk 5 above)
Comment (darwinlopez) [inline src/rg-tag/rg-tag.css]: Done

File: src/rg-tag/rg-tag.css [hunk 13]
```diff
@@ -0,0 +1,45 @@
+/** @define rgTag; use strict */
+
+:root {
+    --Tag-font-size: var(--font-size);
+}
+
+.rgTag {
+  background: var(--color-green-base);
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: sorry can you do the same sort of thing here and with the other colours `--Tag-background-color` for example

File: src/rg-tag/rg-tag.css [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+  font-size: var(--Tag-font-size);
+  margin: 8px 8px 0 0;
+  padding: 0px 12px 0px 12px;
+  text-decoration: none;
+  display: block;
+  height: 32px;
+  line-height: 32px;
+  float: left;
+  box-sizing: border-box;
+  max-width: 100%;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: can you also please sort out the indentation so the whole file is the same please

**Comment by nwhite89** [comment]
@rockabox/qa over to you :+1: :)

**Comment by jim-dj** [comment]
Ready for merge to QA

**Pull request merged**